src/Document.tsx

```diff
@@ -1,7 +1,15 @@
 import {useContext, useMemo, type ReactNode} from 'react'
 import {ChevronDownIcon, ChevronUpIcon, DragHandleIcon} from '@sanity/icons'
-import {AvatarCounter, Card, Box, Button, Flex, Text} from '@sanity/ui'
-import {useSchema, SchemaType, PreviewCard, Preview} from 'sanity'
+import {AvatarCounter, Card, Box, Button, Flex, Text, Tooltip} from '@sanity/ui'
+import {
+  useSchema,
+  SchemaType,
+  PreviewCard,
+  Preview,
+  DocumentStatusIndicator,
+  DocumentStatus,
+  useDocumentVersionInfo,
```

Comment:
If I'm not mistaken the useDocumentVersionInfo-hook were added quite recently – we should probably bump the peer dependency range of `sanity` to the version that introduced it.